docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -158,6 +166,20 @@ just pass -Dquarkus.package.type=native as a property on the command line, howev
 this allows native image tests to also be run.
 
 Create a native executable using: `./mvnw package -Pnative`.
+[[graal-and-windows]]
+[NOTE]
+.Issues with packaging on Windows
+====
+The Microsoft Native Tools for Visual Studio must first be initialized before packaging. You can do this by starting
+the `x64 Native Tools Command Prompt` that was installed with the Visual Studio Build Tools.
+
+Another solution is to write a script to do this for you:
+
+[source,shell]
+----
+cmd /c '"C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Auxiliary\Build\vcvars64.bat" && mvn package -Pnative'
```

Comment:
For VS 2017 the path is `call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvars64.bat"`

Reply:
Will do.

Reply:
@maxandersen Is there a different path for the Windows 7 vars file?

Reply:
@rsvoboda I updated this to `cmd /c 'call "C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Auxiliary\Build\vcvars64.bat" && mvn package -Pnative'` as that's where the file lives when only installing the build tools.

```ps1
C:\Program Files (x86)\Microsoft Visual Studio\2017> gci -r | where name -eq 'vcvars64.bat'


    Directory: C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Auxiliary\Build

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a---           3/18/2020  9:10 AM             39 vcvars64.bat
```